app/code/community/Oyst/OneClick/controllers/NotificationsController.php

```diff
@@ -82,7 +82,9 @@ public function indexAction()
         /** @var Oyst_OneClick_Model_Catalog|Oyst_OneClick_Model_Order $model */
         $response = $model->processNotification($event, $data);
 
-        $this->getResponse()->setHeader('Content-type', 'application/json');
+        if (php_sapi_name() == 'cgi-fcgi') {
```

Comment:
Could you use inverse the test and use === like:
_if ('cgi-fcgi' === php_sapi_name()) {_